content/02.introduction.md

```diff
@@ -1,14 +1,14 @@
 ## Introduction
 
-Knowledge graphs are a practical resource for many real world applications.
-They have been used in social media mining to classify nodes [@doi:10.1007/978-1-4419-8462-3_5] and create recommendation systems [@doi:10.1145/3184558.3186904].
+Knowledge graphs are a practical resource for many real-world applications.
+They have been used in social network mining to classify nodes [@doi:10.1007/978-1-4419-8462-3_5] and create recommendation systems [@doi:10.1145/3184558.3186904].
 These graphs have also been used in natural language processing to interpret simple questions and use relational information to provide answers [@arxiv:1404.4326; @doi:10.1145/3038912.3052675].
-In a biomedical setting these graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
+In a biomedical setting, these graphs have been used to prioritize genes relevant to disease [@pmid:21121028; @doi:10.1101/651000; @doi:10.1109/CIBCB.2019.8791472; @doi:10.1093/bioinformatics/bty559], perform drug repurposing [@doi:10.7554/eLife.26726] and identify drug-target interactions [@doi:10.1371/journal.pcbi.1002574].
 
 Despite their utility, precisely defining a knowledge graph is a difficult task because there are multiple conflicting definitions [@raw:KG_DEF].
-For this review, we define a knowledge graph as the following: a resource that integrates single or multiple sources of information into the form of a graph.
-This graph allows for the capacity to make semantic interpretation, continuously incorporate new information and uncover novel hidden knowledge through computational techniques and algorithms.
-Based on this definition resources like Hetionet [@doi:10.7554/eLife.26726] would be considered a knowledge graph as Hetionet integrates multiple sources of information into the form of a graph (example shown in Figure {@fig:hetionet_schema}).
+For this review, we define a knowledge graph as the following: a resource that integrates single or multiple sources of information into the form of a graph where a node repesents entites and an edge represents a relationship between two entites.
+This graph structure allows for the capacity to make semantic interpretation, continuous incorporation of new information and uncover novel hidden knowledge through computational techniques and algorithms.
```

Comment:
I suspect that "semantic interpretation" is going to depend on how you define a knowledge graph above.